Source/Core/PolylineGeometry.js

```diff
@@ -309,9 +309,11 @@ define([
         var k;
 
         var positions = arrayRemoveDuplicates(polylineGeometry._positions, Cartesian3.equalsEpsilon);
-
         var positionsLength = positions.length;
-        if (positionsLength < 2) {
+
+        // A width of a pixel or less is not a valid geometry, but in order to support external data
+        // that may have errors we treat this as an empty geometry.
+        if (positionsLength < 2 || width < 1.0) {
```

Comment:
A line of width 0.5 works fine and is indeed thinner than a line of width 1.0. Changed this in new commit.